### ISSUE TITLE
Creating new content svg(?) issue

### ISSUE THREAD

**Issue by tesar-tech** [fluentcms/FluentCMS#1839]
![image](https://github.com/user-attachments/assets/d721cdad-339f-475a-bfc6-c794e5234775)

I have this after cloning and running the project.
http://localhost:5000/admin/content-types?pluginId=4da1ddca-11b8-426d-b5c1-56900d20a288&viewName=Create%20Content&slug=blog (but is the same for any content type)

**Comment by TheHadiAhmadi** [comment]
@tesar-tech Is this the content of Rich Text editor (Content field)? or empty page has this problem too?

Can you provide more info? I didn't faced with this problem yet
Thanks

**Comment by tesar-tech** [comment]
Here is what I do: 

https://github.com/user-attachments/assets/f87ce6a4-0d05-4638-8af2-562968c70275

I clone the repo again and the same issue appeared. 



Maybe the issue is related to this warning from quill:

```
quill Overwriting modules/resize with ƒ (e,i){var o,s,n=e.root;function r(){var t=(new(e.getContents().constructor)).retain(1);e.updateContents(t)}n.addEventListener("click",function(t){var e=t.target;t.target&&["img","video"].includes(e.t…
```

Here is the full html, might help: https://gist.github.com/tesar-tech/d538558c907165ab0eddd953c3f59cd4

Brave and FF are facing the same issue on Windows.

**Comment by TheHadiAhmadi** [comment]
@tesar-tech Thanks for details.
This issue was due to a refactor for Sass, which we did 2 days ago.
I've fixed the issue in #1842.
